### PR TITLE
feat: add starknet_l1_to_l2_message_nonce RPC endpoint

### PR DESCRIPTION
**Pull request by dpinones**
* Implement `starknet_l1_to_l2_message_nonce` endpoint
* Add rpc example for `starknet_l1_to_l2_message_nonce`
# Pull Request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build-related changes
- [ ] Documentation content changes
- [ ] Testing
- [ ] Other (please describe):

# What is the current behavior?

Issue Number: #218 

# What is the new behavior?

# Does this introduce a breaking change?

- [ ] Yes
- [x] No

# Other information

### CODE REVIEW

File: crates/beerus-rpc/example_rpc_calls/starknet_l1_to_l2_message_nonce.hurl [hunk 1]
```diff
@@ -0,0 +1,5 @@
+POST http://0.0.0.0:3030
+Content-Type: application/json
+{
+    "jsonrpc":"2.0","method":"starknet_l1_to_l2_message_nonce","params":[],"id":1
```
Comment (drspacemn) [inline crates/beerus-rpc/example_rpc_calls/starknet_l1_to_l2_message_nonce.hurl:4]: Can we add mainnet params that return a value?

**Review by drspacemn — COMMENTED** [review]
I'm getting this result when running beerus-rpc for mainnet:
```
{"jsonrpc":"2.0","result":"0x0","id":1}
```
but should be returning > 180124 ?

https://etherscan.io/address/0xc662c410c0ecf747543f5ba90660f6abebd9c8c4#readProxyContract#F8

**Comment by dpinones** [comment]
> I'm getting this result when running beerus-rpc for mainnet:
> 
> ```
> {"jsonrpc":"2.0","result":"0x0","id":1}
> ```
> 
> but should be returning > 180124 ?
> 
> https://etherscan.io/address/0xc662c410c0ecf747543f5ba90660f6abebd9c8c4#readProxyContract#F8

It can be tested by setting the `STARKNET_CORE_CONTRACT_ADDRESS` environment variable with the mainnet contract `0xc662c410C0ECf747543f5bA90660f6ABeBD9C8c4`.

Resp:
```
{
  "jsonrpc": "2.0",
  "result": "0x2bf9b",
  "id": 1
}
```

See https://github.com/starknet-io/starknet-addresses for the StarkNet core contract address on different networks.
This is [where](https://github.com/keep-starknet-strange/beerus/blob/ba500a95dca96f67229c943cd8db60315585abf1/crates/beerus-core/src/config.rs#L45) you read the `STARKNET_CORE_CONTRACT_ADDRESS` environment variable

**Comment by drspacemn** [comment]
Thank you, mine was commented out!

LGTM

**Pull request merged**